<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stick20HiddenVolumeDialog</class>
 <widget class="QDialog" name="stick20HiddenVolumeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setup hidden volume</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/CS_icon.png</normaloff>:/images/CS_icon.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Hidden volume slot 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hidden volume slot 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hidden volume slot 3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hidden volume slot 4</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="HVPasswordEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>12345678901234567890</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Start at % of SD size</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>End at % of SD size</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="StartBlockSpin">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="EndBlockSpin">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="ShowPasswordCheckBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Show password</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>stick20HiddenVolumeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>stick20HiddenVolumeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
